src/components/Agreement/AgreementHeader.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { IconCheck, Tag, textStyle, useLayout, useTheme, GU } from '@aragon/ui'
+import icon from './assets/icon.svg'
+
+function AgreementHeader({ title }) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const compactMode = layoutName === 'small'
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        margin-bottom: ${compactMode ? 4 * GU : 5 * GU}px;
+      `}
+    >
+      {!compactMode && <DecorativeIcon />}
+      <div
+        css={`
+          display: flex;
+          flex: 1;
+          justify-content: space-between;
+        `}
+      >
+        <div>
+          <h2
+            css={`
+              ${compactMode ? textStyle('title3') : textStyle('title2')};
+              margin-bottom: ${0.75 * GU}px;
+            `}
+          >
+            {title}
+          </h2>
+          <div
+            css={`
+              display: flex;
+            `}
+          >
+            <Tag
```

Comment:
Just checking, this tag should always appear? Or do we need a validation?

Reply:
Good question, I think for the time being this can't enter any other state, there was discussion around whether we really need the tag at all but I do think it helps and is a nice piece of reassurance.